### PR TITLE
Release 0.7.0

### PR DESCRIPTION
**Pull request by End-rey**

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-v0.5.0
+v0.7.0
```
Comment (roman-khimov) [inline VERSION:1]: I think this file can just be dropped (along with Makefile uses of it), it's just a library now.

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,23 @@ Changelog for NeoFS LOCODE database
 
 ## [Unreleased]
 
+## [0.7.0] - 2024-08-28
+
+### Changed
+- Go 1.22 is required to build now (#40, #43)
+- Added a file with which you can overwrite the coordinates of incorrect 
+records (#38)
```
Comment (roman-khimov) [inline CHANGELOG.md:11]: Usually lines are not breaked here, if they are then you're missing two leading spaces on this line.

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,23 @@ Changelog for NeoFS LOCODE database
...
+- Drop cobra dependency, so there is no CLI (#42)
+
+### Updated
+- UN/LOCODE to `2024-1` version (#44)
+- UN/LOCODE files from gitHub (#37)
+- Continents file from known source (#45)
+
+### Fixed
+- UN/LOCODE files now in Utf-8, without wrong symbols (#37)
```
Comment (roman-khimov) [inline CHANGELOG.md:20]: UTF-8

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,23 @@ Changelog for NeoFS LOCODE database
 
 ## [Unreleased]
 
+## [0.7.0] - 2024-08-28
+
+### Changed
+- Go 1.22 is required to build now (#40, #43)
+- Added a file with which you can overwrite the coordinates of incorrect
```
Comment (roman-khimov) [inline CHANGELOG.md:10]: It's more like "corrected known bad coordinates via a local override".

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,23 @@ Changelog for NeoFS LOCODE database
...
+
+### Changed
+- Go 1.22 is required to build now (#40, #43)
+- Added a file with which you can overwrite the coordinates of incorrect 
+records (#38)
+- Drop cobra dependency, so there is no CLI (#42)
+
+### Updated
+- UN/LOCODE to `2024-1` version (#44)
+- UN/LOCODE files from gitHub (#37)
```
Comment (roman-khimov) [inline CHANGELOG.md:16]: More like 'switch data source to github.com/datasets/un-locode which has more suitable data format'.

**Review by roman-khimov — COMMENTED** [review]
But why ` CHANGELOG: delete VERSION `? It's not a changelog change.

File: CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,22 @@ Changelog for NeoFS LOCODE database
 
 ## [Unreleased]
 
+## [0.7.0] - 2024-08-28
+
+### Changed
+- Go 1.22 is required to build now (#40, #43)
+- Corrected known bad coordinates via a local override (#38)
+- Drop cobra dependency, so there is no CLI (#42)
```
Comment (carpawell) [inline CHANGELOG.md:11]: > there is no CLI

i would not understand it as an external reader. cli is the same and we still have it but it uses standard library now

**Review by roman-khimov — APPROVED** [review]

**Review by carpawell — APPROVED** [review]

**Pull request merged**